Gopkg.lock

```diff
@@ -73,17 +73,23 @@
   version = "v0.2.0"
 
 [[projects]]
-  branch = "master"
-  name = "github.com/google/subcommands"
+  name = "github.com/inconshreveable/mousetrap"
   packages = ["."]
-  revision = "a3682377147edf596d303faabd89f81977b3f678"
+  revision = "76626ae9c91c4f2a10f34cad8ce83ea42c93bb75"
+  version = "v1.0"
 
 [[projects]]
   name = "github.com/pkg/errors"
   packages = ["."]
   revision = "645ef00459ed84a119197bfb8d8205042c6df63d"
   version = "v0.8.0"
 
+[[projects]]
+  name = "github.com/spf13/cobra"
```

Comment:
It'll save me / you merge headaches if you copy the cobra bit from the Gopkg.toml in #102